user/views.py

```diff
@@ -130,8 +132,11 @@ def destroy(self, request, *args, **kwargs):
 
     @action(detail=True)
     def query(self, request, username=None):
-        return Response(
-            success=len(username) <= 150
-            and username.lower() not in FORBIDDEN_USERNAMES
-            and not SluglineUser.objects.filter(username=username).exists()
-        )
+        if (
+            SluglineUser.objects.filter(username=username).exists()
+            or username.lower() in FORBIDDEN_USERNAMES
+        ):
+            raise ValidationError({"username": "USER.ALREADY_EXISTS"})
+        if len(username) > 150:
+            raise ValidationError({"username": "USER.TOO_LONG"})
```

Comment:
Two issues: 

1. The correct namespace is a tad longer than just `USER`; it's `USER.USERNAME`
2. Errors should be an array, e.g. `{"username": ["USER.USERNAME.TOO_LONG"]}`

Reply:
Done and done. https://github.com/UWmathNEWS/slugline-api/pull/6/commits/a41d7cc9feb587187bad50560ed4fffd09c6d294